app/src/main/java/dadeindustries/game/gc/mechanics/turn/TurnProcessor.java

```diff
@@ -61,7 +64,7 @@
 		return events;
 	}
 
-	private ArrayList<Spaceship> getAllShipsForPlayer(Sector[][] sectors, Player player) {
+	private static ArrayList<Spaceship> getAllShipsForPlayer(Sector[][] sectors, Player player) {
```

Comment:
Did the guy teaching you java in first year go on about how you have to ask yourself why every time you type `static`?